core/src/smtp/hotmail.rs

```diff
@@ -193,4 +251,12 @@ mod tests {
 		let f = f1.try_join(f2).await;
 		assert!(f.is_ok(), "{:?}", f);
 	}
+
+	#[test]
```

Comment:
No tests for the `check_outlook_api()` function—any idea if there's a `someone@gmail.com` equivalent that could be used here?